### DISCUSSION TITLE
With nginx proxy root path

### DISCUSSION THREAD

**Discussion by NickLin910221** [open-webui/open-webui#3720] [General]
I'm struggled with using nginx proxy with suburl
Like nginx(https://doamin/chatbot)->open-webui
![telegram-cloud-photo-size-5-6079899436307300552-y](https://github.com/open-webui/open-webui/assets/39323130/9605c240-5ed0-4e25-a2a1-421b654dad7e)
How to achieve it.

**Comment by gizbo** [comment]
Is there a variable that can be set to modify the expected application root context ? 
Thanks,

**Reply by entropy2333** [reply]
It might disappoint you, this feature is currently not implemented. For more information, please refer to https://github.com/open-webui/open-webui/discussions/3431. Repackaging the Docker image might be feasible.
